ovos_dialog_normalizer_plugin/util.py

```
@@ -0,0 +1,718 @@
+
+    # Portuguese dates and times
+    print("\n--- Portuguese Date & Time Examples ---")
+    print(f"Portuguese date (A data é 03/08/2025): {normalize('A data é 03/08/2025', 'pt')}")
+    print(
+        f"Portuguese ambiguous date (O relatório é para 15/05/2025): {normalize('O relatório é para 15/05/2025', 'pt')}")
+    print(
+        f"Portuguese date with dashes (O evento é no dia 25-10-2024): {normalize('O evento é no dia 25-10-2024', 'pt')}")
+    print(f"Portuguese military time (O encontro é às 14h30): {normalize('O encontro é às 14h30', 'pt')}")
+
+    # Other examples
+    print(f"\n--- Other Examples ---")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)